integration/helpers.go

```diff
@@ -77,6 +78,218 @@ const (
 	Host     = "localhost"
 )
 
+const (
+	HostID = "00000000-0000-0000-0000-000000000000"
+	Site   = "local-site"
+)
+
+type integrationTestSuite struct {
```

Comment:
This isn't used for all tests, is it?

If not, can we give it a more specific name?